tests/cpp/test_alias.cpp

```diff
@@ -520,6 +520,9 @@ TEST_F(AliasTest, AliasOutputBeforeNonAliasOutput) {
   testValidate(
       executor_cache.fusion(), out_tensors, {in_tensor}, __LINE__, __FILE__);
 
+  // TODO: Fix the alias support
```

Comment:
This is broken for now. Need to understand how it actually works before this PR.

Reply:
#3572